media/js/firefox/new/experiment-features-variation.js

```diff
@@ -0,0 +1,74 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    var featuresVarA = document.querySelectorAll('.features ul > li');
+    var featuresVarC = document.querySelectorAll('.c-card-feature');
+
+    function trackFeatureClick(e) {
```

Comment:
Note for reviewer: there is an ask in this experiment to track how many clicks each feature gets, despite the features not being interactive in any way. I tried to convey that this is pretty unlikely to get much data as there is nothing to indicate that these things are clickable, but the requesting team is is adamant that they still want this. I also tried to suggest other ways to measure engagement, such as scroll tracking, but couldn't persuade them to change their minds.